From 2cf9b4339f12cdcb17d234c63af35e27a21ce350 Mon Sep 17 00:00:00 2001
From: Meron Cohen <mecohen@habana.ai>
Date: Sun, 18 May 2025 09:37:56 +0300
Subject: [PATCH] [SW-229188] g2e: set mme pmu rls

MME PMU RLs were set for G2D only at first.
It was confirmed that G2E limitations should be on top of G2D limitations.
Therefore, setting these RLs for G2D and G2E (G2D and above).

Change-Id: Idba2fcc0c0a58f63607c0a00e000239416593fe4
Signed-off-by: Meron Cohen <mecohen@habana.ai>
(cherry picked from commit 3ca20cfbcb956371927e8f60d7e0ed57b4178837)
---

diff --git a/src/gaudi2/engines.c b/src/gaudi2/engines.c
index ef23b61..1feada8 100644
--- a/src/gaudi2/engines.c
+++ b/src/gaudi2/engines.c
@@ -40,7 +40,7 @@
 {
 	unsigned int i;
 
-	if (get_bis_revision() != GAUDI2_BIS_REV_D)
+	if (get_bis_revision() < GAUDI2_BIS_REV_D)
 		return;
 
 	for (i = 0; i < NUM_OF_DCORES; i++) {
